src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -310,7 +310,11 @@ void SmartScript::ProcessAction(SmartScriptHolder& e, Unit* unit, uint32 var0, u
             {
                 if (IsUnit(target))
                 {
-                    target->PlayDirectSound(e.action.sound.sound, e.action.sound.onlySelf ? target->ToPlayer() : nullptr);
+                    if (e.action.sound.distance = 1)
```

Comment:
Double equal sign `==` instead of single `=` here, please.

Reply:
Oops, sorry was tired yesterday, ill fix it this evening.